services/certmanager/resources/certificates.go

```diff
@@ -0,0 +1,71 @@
+package resources
+
+import (
+	"context"
+
+	sdkgo "github.com/ionos-cloud/sdk-go-cert-manager"
+)
+
+type Response struct {
+	sdkgo.APIResponse
+}
+
+// CertsService is a wrapper around ionoscloud.CertificateDto
+type CertsService interface {
+	Get(certId string) (sdkgo.CertificateDto, *sdkgo.APIResponse, error)
+	Post(sdkgo.CertificatePostDto) (sdkgo.CertificateDto, *sdkgo.APIResponse, error)
+	List() (sdkgo.CertificateCollectionDto, *sdkgo.APIResponse, error)
+	Delete(certId string) (*sdkgo.APIResponse, error)
+	Patch(certId string, input sdkgo.CertificatePatchDto) (sdkgo.CertificateDto, *sdkgo.APIResponse, error)
+	GetApiVersion() (sdkgo.ApiInfoDto, *sdkgo.APIResponse, error)
+}
+
+type certsService struct {
+	client  *Client
+	context context.Context
+}
+
+var _ CertsService = &certsService{}
+
+func NewCertsService(client *Client, ctx context.Context) CertsService {
+	return &certsService{
+		client:  client,
+		context: ctx,
+	}
+}
+
+func (svc *certsService) Get(certId string) (sdkgo.CertificateDto, *sdkgo.APIResponse, error) {
+	req := svc.client.CertificatesApi.CertificatesGetById(svc.context, certId)
+	cert, res, err := svc.client.CertificatesApi.CertificatesGetByIdExecute(req)
+	return cert, res, err
```

Comment:
This function can be one line I think : return svc.client.CertificatesApi.CertificatesGetById(svc.context, certId).Execute()

Reply:
@cristiGuranIonos I agree with this but this is the case for every single service in IonosCTL

To be frank, I don't see a good reason for having services, seeing as the SDKs we import basically act as a service layer anyway

The only thing these services manage to accomplish is generate more "paperwork" for the developer IMHO

Seeing as this problem permeates into the entirety of the codebase, what do you think of handling in a different PR @cristiGuranIonos ?

Reply:
yes, definitely. As part of a refactor.